pyvista/jupyter/pv_ipygany.py

```diff
@@ -194,7 +194,8 @@ def show_ipygany(plotter, return_viewer, height=None, width=None):
 
     if cbar is not None:
         # Colormap choice widget
-        colormap_dd = Dropdown(options=colormaps, description='Colormap:')
+        options = [(key, value) for key, value in colormaps.items()]
```

Comment:
This is the same as `options = list(colormaps.items())`?

Reply:
```suggestion
        options = list(colormaps.items())
```